masschallenge/django-accelerator
# AC-4669: More standard mechanism for tests and making migrations

To test:
- Run tests with `make test`
- Change a model and run `make migrations` 
- Install tox with `pip install tox` and run `tox`

Confirm that they all behave as expected.
Review code changes.
